.github/workflows/ci.yml

```diff
@@ -213,7 +213,7 @@ jobs:
             !presto-mongodb,!presto-kafka,!presto-elasticsearch,
             !presto-redis,
             !presto-sqlserver,!presto-postgresql,!presto-mysql,!presto-memsql,
-            !presto-oracle,
+            !presto-oracle,!presto-vertica,
```

Comment:
Why do we bundle oracle with vertica?

Reply:
No special reason, but the CI time by bundling with oracle will be almost the same as the above step (sqlserver~memsql)